### ISSUE TITLE
Version Incompatibility and Installation Issue

### ISSUE THREAD

**Issue by chaojian-zhang** [kensk8er/chicksexer#12]
Currently there is no way to use `chicksexer` in either **Python 3.5.0** and **Python 3.8.2**.

For the latter case, I mentioned it in [this comment](https://github.com/kensk8er/chicksexer/issues/10#issuecomment-795724854). To recap, you will be able to successfully install `chicksexer` in **Python 3.8.2** but will receive this error: `ModuleNotFoundError: No module named 'tensorflow.contrib'`. I think this is due to a naming changing in [Tensorflow 2](https://github.com/tensorflow/tensorflow/issues/31350). The obvious workaround was to use Python 3.5.0.

However, here I am noting that **even with Python 3.5.0**, the solution given in [this issue](https://github.com/kensk8er/chicksexer/issues/10#issue-709555195) still doesn't work. I am receiving the following error: 

```
Collecting scipy>=0.18.1 (from chicksexer)
  Downloading https://files.pythonhosted.org/packages/26/68/84dbe18583e79e56e4cee8d00232a8dd7d4ae33bc3acf3be1c347991848f/scipy-1.6.1.tar.gz (27.3MB)
    100% |################################| 27.3MB 24kB/s
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 20, in <module>
      File "C:\Users\szinu\AppData\Local\Temp\pip-build-12wmumib\scipy\setup.py", line 31, in <module>
        raise RuntimeError("Python version >= 3.7 required.")
    RuntimeError: Python version >= 3.7 required.
```

I suppose this has something to do with this pull request: https://github.com/kensk8er/chicksexer/pull/11

In other words, currently, there is no way to run `chicksexer` at all!

**Comment by chaojian-zhang** [comment]
I tried to uninstall my python and reinstall `chicksexer` and I am getting a different error: 

```
Could not find a version that satisfies the requirement tensorflow==1.0.1
```

**Comment by chaojian-zhang** [comment]
Good news! I finally managed to install `chicksexer` with the following setup:

1) I used `pyenv-win` to create a local **Python 3.5.2** version (3.5.0 doesn't work)
2) Use `venv` to create a local virtual environment
3) Then gradually apply `python -m pip install --upgrade pip<=15.0`,  `python -m pip install --upgrade pip<=18.0`,  `python -m pip install --upgrade pip<=19.0` and  `python -m pip install --upgrade pip` to get to the latest version of `pip` (otherwise you might get some weird errors of broken pip).
4) Install `chicksexer` using `pip install chicksexer`.
5) Works great!

**Comment by BYUJason** [comment]
@chaojian-zhang - a bit of a noob at this but trying it out since I'd really like to use this package. I'm using windows cmd line.

I think I'm getting that broken pip error "sys.stderr.write(f"ERROR: {exc}")"

I tried installing gradually python -m pip install --upgrade pip<=15.0 but it says "The system cannot find the file specified."

Any ideas?

**Comment by chaojian-zhang** [comment]
@BYUJason what's your current pip version?
